modules/http/src/main/java/org/mule/module/http/internal/listener/grizzly/GrizzlyServerManager.java

```diff
@@ -186,7 +189,7 @@ public Server createSslServerFor(TlsContextFactory tlsContextFactory, WorkManage
         sslFilterDelegate.addFilterForAddress(serverAddress, createSslFilter(tlsContextFactory));
         httpServerFilterDelegate.addFilterForAddress(serverAddress, createHttpServerFilter(usePersistentConnections, connectionIdleTimeout));
         executorProvider.addExecutor(serverAddress, workManagerSource);
-        final GrizzlyServer grizzlyServer = new GrizzlyServer(serverAddress, transport, httpListenerRegistry);
+        final GrizzlyServer grizzlyServer = wrap(new GrizzlyServer(serverAddress, transport, httpListenerRegistry), serverAddress);
```

Comment:
Review these changes with @pablolagreca to see is there is a simpler alternative approach with some changes to http connector.